<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BATTLE CITY</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>146</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="startLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Select difficulty</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="easyPushButton">
      <property name="text">
       <string>Easy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mediumPushButton">
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hardPushButton">
      <property name="text">
       <string>Hard</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>133</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="gameOverLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Game over</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="restartButton">
      <property name="text">
       <string>Restart</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>160</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Health:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="healthLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Kills:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="killsLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
